src/graph/query_graph.c

```diff
@@ -171,11 +173,11 @@ static void _QueryGraph_ExtractNode
 
 			QueryGraph_AddNode(graph, n);
 			// set node label information
-			_QueryGraphSetNodeLabel(n, ast_node);
+			_QueryGraphSetNodeLabel(n, ast_node, false);
 		}
 	} else {
 		// set node label information
-		_QueryGraphSetNodeLabel(n, ast_node);
+		_QueryGraphSetNodeLabel(n, ast_node, false);
```

Comment:
add comment explain how we get to this

Reply:
Reached when we extract a node which has already been introduced to graph (the graph to which we copy the original  QueryGraph). Same as above.